flexmeasures/cli/__init__.py

```diff
@@ -13,3 +15,15 @@ def register_at(app: Flask):
             import flexmeasures.cli.data_delete
             import flexmeasures.cli.db_ops
             import flexmeasures.cli.testing  # noqa: F401
+
+
+def is_running() -> bool:
+    """
+    True if we are running one of the custom FlexMeasures CLI commands.
+    """
+    cli_sets = ("add", "delete", "show", "monitor", "jobs", "db-ops")
+    command_line = " ".join(sys.argv)
+    for cli_set in cli_sets:
+        if f"flexmeasures {cli_set}" in command_line:
```

Comment:
Could this be more strict by ensuring that `flexmeasures {cli_set}` occurs at the start of the command and not somewhere in the middle?

Reply:
No. Here is a typical value of `command_line`:

```
/home/nicolas/envs/sempras/bin/flexmeasures run
```